### ISSUE TITLE
RenderManagerConfig windowTitle issue

### ISSUE THREAD

**Issue by JeroMiya** [OSVR/OSVR-Core#282]
Sometimes when a ```RenderManagerConfig``` is parsed (via ```createShared```), the windowTitle field is corrupted somehow. I'm getting this when OSVR-Unreal is instantiating a render manager instance, which internally creates a ```RenderManagerConfig```. I'm not sure if the issue is a cross-DLL C++ issue or something else.

**Comment by russell-taylor** [comment]
This is probably a race condition.  The RenderManager waits until it gets a display and then presumes that the rendermanager config is fine.  If you know how to make it wait on both of them, go ahead update.  Otherwise I'll try and figure it out.

**Comment by russell-taylor** [comment]
Did this get fixed by the recent commit, or was that only a work-around?

**Comment by JeroMiya** [comment]
If there is a race condition with the display/rendermanagerconfig then we should file that as a separate issue - this seems to happen even when both are available at the time it attempts to parse.

See this pull request:
https://github.com/OSVR/OSVR-Core/pull/296

Fixes this and other cross-DLL ABI issues by making RenderManagerConfig a header-only API. The RenderManager dll is currently the only consumer of this header, though I would like to see it used in OSVR-Core when we sync projection and DisplayConfig APIs between core and render manager.

**Comment by JeroMiya** [comment]
Fixed with https://github.com/OSVR/OSVR-Core/pull/296
